### PR TITLE
Use consistent dualToR terms 'upper_tor' and 'lower_tor'

### PR DESCRIPTION
**Pull request by wangxin**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
The mux simulator server and client code uses term "tor_a" and "tor_b" refering the
upper ToR and lower ToR. To be consistent across documentations and other codes,
this change updated them to "upper_tor" and "lower_tor".

#### How did you do it?
In mux simulator server, client and documentation:
* updated term "tor_a" to "upper_tor".
* updated term "tor_b" to "lower_tor".

### CODE REVIEW

**Review by bingwang-ms — APPROVED** [review]

**Comment by bingwang-ms** [comment]
Thanks for the update!

File: ansible/roles/vm_set/files/mux_simulator.py [hunk 1]
```diff
@@ -22,6 +22,11 @@
 app = Flask(__name__)
 
 
+TOR_A = 'upper_tor'
```
Comment (theasianpianist) [inline ansible/roles/vm_set/files/mux_simulator.py:25]: Could we also change these variable names?
```suggestion
UPPER_TOR = 'upper_tor'
```

File: ansible/roles/vm_set/files/mux_simulator.py [hunk 2]
```diff
@@ -22,6 +22,11 @@
...
+TOR_B = 'lower_tor'
```
Comment (theasianpianist) [inline ansible/roles/vm_set/files/mux_simulator.py:26]:
```suggestion
LOWER_TOR = 'lower_tor'
```

File: tests/common/dualtor/mux_simulator_control.py [hunk 3]
```diff
@@ -7,8 +7,8 @@
 
 logger = logging.getLogger(__name__)
 
-TOR_A = "tor_a"
-TOR_B = "tor_b"
+TOR_A = "upper_tor"
```
Comment (theasianpianist) [inline tests/common/dualtor/mux_simulator_control.py:10]:
```suggestion
UPPER_TOR = "upper_tor"
```

File: tests/common/dualtor/mux_simulator_control.py [hunk 4]
```diff
@@ -7,8 +7,8 @@
...
+TOR_B = "lower_tor"
```
Comment (theasianpianist) [inline tests/common/dualtor/mux_simulator_control.py:11]: LOWER_TOR

Comment (wangxin) [reply tests/common/dualtor/mux_simulator_control.py:11]: Sure, I'll update.

**Comment by wangxin** [comment]
retest vsimage please

**Pull request merged**